docs/workflow/building/libraries/cross-building.md

```diff
@@ -24,108 +23,102 @@ and for arm64 you need:
+To build managed part of libraries for arm using subset (architecture-dependent, can't be used on other architectures):
 
-The output is at `artifacts/bin/[BuildSettings]` where `BuildSettings` looks something like `net5.0-<TargetOS>-Debug-<Architecture>`. Ex: `artifacts/bin/net5.0-Linux-Debug-x64`. For more details on the build configurations see [project-guidelines](/docs/coding-guidelines/project-guidelines.md)
+    $ ./build.sh --arch arm --librariesConfiguration Release --subset libs.sfx
 
-Building corefx for Linux ARM Emulator
-=======================================
+Note that by default ILLinker trimming is enabled and libraries built above for arm can't be used on other arches. To build architecture-independent managed part of libraries for arm using subset:
 
-It is possible to build corefx binaries (native and managed) for the Linux ARM Emulator (latest version provided here: [#5394](https://github.com/dotnet/runtime/issues/5394)).
-The `scripts/arm32_ci_script.sh` script does this.
+    $ ./build.sh --arch arm --librariesConfiguration Release --subset libs.sfx /p:ILLinkTrimAssembly=false
 
-The following instructions assume that:
-* You have set up the extracted emulator at `/opt/linux-arm-emulator` (such that `/opt/linux-arm-emulator/platform/rootfs-t30.ext4` exists)
-* The mount path for the emulator rootfs is `/opt/linux-arm-emulator-root` (change this path if you have a working directory at this path).
+Build artifacts can be found in `artifacts/bin/microsoft.netcore.app.runtime.<TargetOS>-<BuildArch>/<BuildType>/runtimes/<TargetOS>-<BuildArch>/lib/net7.0/`. For more details on the build configurations see [project-guidelines](/docs/coding-guidelines/project-guidelines.md).
 
-All the following instructions are for the Release mode. Change the commands and files accordingly for the Debug mode.
+Both native and managed parts can be built at the same time with:
 
-To just build the native and managed corefx binaries for the Linux ARM Emulator, run the following command:
-```
-prajwal@ubuntu ~/corefx $ ./scripts/arm32_ci_script.sh \
-    --emulatorPath=/opt/linux-arm-emulator \
-    --mountPath=/opt/linux-arm-emulator-root \
-    --buildConfig=Release
-```
+    $ ROOTFS_DIR=`pwd`/.tools/rootfs/arm ./build.sh --cross --arch arm --librariesConfiguration Release --subset libs.native+libs.sfx
 
-The Linux ARM Emulator is based on the soft floating point and thus the native binaries are generated for the armel architecture. The corefx binaries generated by the above command can be found at `~/corefx/artifacts/bin/Linux.armel.Release`, `~/corefx/artifacts/bin/Linux.AnyCPU.Release`, `~/corefx/artifacts/bin/Unix.AnyCPU.Release`, and `~/corefx/artifacts/bin/AnyOS.AnyCPU.Release`.
+Build libraries for a new architecture
+===================================
 
+When building for a new architecture you will need to build the native pieces separately from the managed pieces in order to correctly boot strap the native runtime.
```

Comment:
I haven't noticed any special care for new architectures we recently ported (armv6, loongarch64, riscv64)?

Reply:
Yes, the platform configs are centralized. Once we start porting to new platform, the first few files we update are in `eng/native/`, `eng/versioning.targets` and `Directory.Build.props` file at repo's root. Most msbuild and shell scripts inherit configs from those files.

Reply:
What I meant was that previously libs build downloaded nuget packages specific to arch/platform, but since new platform didn't yet have these nuget packages, we had to build for x64. Maybe now this is not the case.

Reply:
Yup, I think it wsa changed since the switch to `live` builds (in the early days of dotnet/runtime repo).

Reply:
I removed this whole section